### PR TITLE
add drag&drop feature

### PR DESCRIPTION
**Pull request by spacewoox**
add drag and drop feature with flow.js in ember front application

### CODE REVIEW

File: nanocloud/main.go [hunk 1]
```diff
@@ -91,6 +91,26 @@ func main() {
 	e.Use(middleware.Recover())
 
 	/**
+	* CORS Policy
```
Comment (willrnch) [inline nanocloud/main.go:94]: remove this

File: nanocloud/main.go [hunk 2]
```diff
@@ -91,6 +91,26 @@ func main() {
...
+	 */
+	e.Use(func(h echo.HandlerFunc) echo.HandlerFunc {
```
Comment (dynamiccast) [inline nanocloud/main.go:96]: This should not be commited

File: webapp/app/components/drag-n-drop.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+
+  that: this,
+	token: null,
```
Comment (dynamiccast) [inline webapp/app/components/drag-n-drop.js:6]: Indent issue

File: webapp/app/components/drag-n-drop.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+  drop() {
+  },
+
+  initDropZone: function(){
+
+    var that = this.that;
+
+		var flow = new Flow({
+      target: 'http://localhost:8080/upload',
```
Comment (dynamiccast) [inline webapp/app/components/drag-n-drop.js:33]: Hardcoded

File: webapp/app/components/drag-n-drop.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  dragLeave(e) {
+    this.hideElement();
+  },
+
+  drop() {
+  },
+
+  initDropZone: function(){
+
+    var that = this.that;
```
Comment (dynamiccast) [inline webapp/app/components/drag-n-drop.js:30]: Statement with no effect

File: webapp/app/components/drag-n-drop.js [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+		var flow = new Flow({
+      target: 'http://localhost:8080/upload',
+			headers: { Authorization: "Bearer YhcWIstQBdULmysDQXXkACayL" },
+			singleFile: true
+		});
+
+		flow.assignDrop(this.element);
+
+		flow.on('filesSubmitted', function(){
+      console.log('submit file');
```
Comment (dynamiccast) [inline webapp/app/components/drag-n-drop.js:41]: Please remove debug log

**Pull request closed without merging**